server/flow.go

```diff
@@ -228,9 +232,12 @@ func (fm *FlowManager) stepServerVersionQuestion() flow.Step {
 	return flow.NewStep(stepServerVersionQuestion).
 		WithText(delegateQuestionText).
 		WithButton(flow.Button{
-			Name:    "Yes",
-			Color:   flow.ColorPrimary,
-			OnClick: flow.Goto(stepCSversionGreaterthan9),
+			Name:  "Yes",
+			Color: flow.ColorPrimary,
+			OnClick: func(f *flow.Flow) (flow.Name, flow.State, error) {
+				fm.plugin.serverVersionGreaterthan9 = true
```

Comment:
Is there a way for us to detect the server version using the API?

Reply:
I could not find a way to get this from API